app/views/shared/views/_show_containers_dashboard.html.haml

```diff
@@ -78,7 +78,7 @@
                    "pf-utilization-trend-chart" => "",
                    "sparkline-config"           => "cpuUsageSparklineConfig"}
             %span.trend-footer-pf{"ng-class" => "{ 'chart-transparent-text': !cpuUsageData }"}
-              = _("Last 30 Days")
+              = _("{{cpuUsageConfig.timeFrame}}")
```

Comment:
No point in translating `"{{cpuUsageConfig.timeFrame}}"` .. the only thing this can accomplish is that the translators will break your code ;).

If the actual string is defined in the frontend, you can't very well translate in on the backend..